src/Util/SiteConfig.pm

```diff
@@ -0,0 +1,19 @@
+use strict;
+
+package Util::SiteConfig;
+use Exporter::Easy (EXPORT => [ '%config' ]);
+
+use JSON;
+
+use vars qw(%config);
+
+{
+  my $config_fn = 'config.json';
```

Comment:
Reading the config from the current working directory seems problematic for the cronjobs. I'd prefer for this to be an absolute pathname read from an environment variable (and die-ing with an error if the environment variable is not set). The environment variable could be defaulted in app.fcgi if not set in the Apache config, something like:

TM_CONFIG=${TM_CONFIG:-$(dirname $0)/config.json}